enumeratum-core/src/test/scala/enumeratum/EnumSpec.scala

```diff
@@ -250,7 +250,28 @@ class EnumSpec extends FunSpec with Matchers {
         case object Bar extends Foo
         case object Baz extends Foo
       }
-        """ shouldNot compile
+      """ shouldNot compile
+    }
+  }
+
+  describe("in") {
+    import DummyEnum._
+
+    it("should return true if enum value is contained by the parameter list") {
+      val enum: DummyEnum = Hello
```

Comment:
Right, something similar happens with integrations with [libs like Slick](https://github.com/lloydmeta/enumeratum#slick-integration). Not a problem :+1: